### PR TITLE
feat: aau i access my account profile

### PR DESCRIPTION
**Pull request by stanleyyconsensys**
## Description

The PR is to :

- Adding a edit button and visible only if account is connected and match the page address

- When connected, clicking on ConnectButton on the top bar will redirect to my account page

image

### Related ticket
#2 
Fixes #

### Type of change

- [ ] Chore
- [ ] Bug fix
- [X] New feature
- [ ] Documentation update

### CODE REVIEW

**Review by dawnseeker8 — APPROVED** [review]
finish reviewing.

**Review by satyajeetkolhapure — APPROVED** [review]

File: src/components/ConnectButton.test.tsx [hunk 1]
```diff
@@ -1,10 +1,43 @@
...
+import { navigate } from 'gatsby';
+
 import { ConnectButton } from './ConnectButton';
+// eslint-disable-next-line jest/no-mocks-import
+import { mockGetIsConnected, mockShowModel } from '../__mocks__/connectkit';
 import { render } from '../utils/test-utils';
 
 describe('ConnectKitButton.Custom', () => {
-  it('renders', () => {
+  it('renders', async () => {
```
Comment (satyajeetkolhapure) [inline src/components/ConnectButton.test.tsx:10]: is this async required?

File: src/__mocks__/connectkit.tsx [hunk 2]
```diff
@@ -21,16 +21,20 @@ export type MockConnectKitButtonCustomProps = {
...
   return (
     <div className="mock-connectkit-button-custom">
       {children({
-        isConnected: mockIsConnected,
-        show: jest.fn(),
+        isConnected: mockGetIsConnected(),
+        show: mockShowModel,
         truncatedAddress: '0x...',
-        ensName: mockEnsName,
+        ensName: 'fake.ens.name',
```
Comment (satyajeetkolhapure) [inline src/__mocks__/connectkit.tsx:37]: just one suggestion, not major, in naming fake will give an impression of invalid, unless it is meant to be. mock.ens.name might suite better

**Review by satyajeetkolhapure — APPROVED** [review]

File: src/__mocks__/connectkit.tsx (hunk 2 above)
Comment (stanleyyconsensys) [reply src/__mocks__/connectkit.tsx:37]: good point

File: src/components/ConnectButton.test.tsx (hunk 1 above)
Comment (stanleyyconsensys) [reply src/components/ConnectButton.test.tsx]: fixed

File: src/__mocks__/connectkit.tsx (hunk 2 above)
Comment (stanleyyconsensys) [reply src/__mocks__/connectkit.tsx]: fixed

**Review by satyajeetkolhapure — APPROVED** [review]

**Review by ritave — APPROVED** [review]

**Pull request merged**